app/stac_api/serializers.py

```diff
@@ -514,23 +515,37 @@ class Meta:
 
     # NOTE: when explicitely declaring fields, we need to add the validation as for the field
     # in model !
+    item = serializers.SlugRelatedField(
+        slug_field='name', write_only=True, queryset=Item.objects.all()
+    )
+    id = serializers.CharField(
+        source='name',
+        required=True,
+        max_length=255,
+        validators=[validate_name, UniqueValidator(queryset=Asset.objects.all())]
+    )
+    title = serializers.CharField(max_length=255, required=False, allow_blank=True)
     type = serializers.CharField(source='media_type', max_length=200)
     # Here we need to explicitely define these fields with the source, because they are renamed
     # in the get_fields() method
+    description = serializers.CharField(max_length=255, required=False, allow_blank=True)
```

Comment:
I would not add the `description` explicitly here, it is not needed. Note that when adding explicitly fields we need to make sure that the validation is done as in the model ! See comments.

Reply:
I made them explicit because I assumed the default value for `required` would be `True` and we want `False` here.
But I did not test, if they default to `False` automatically, as the model defines `blank` or `null` as `True`. 
As discussed, I'll add unit test first, and once they pass successfully, I'll start deleting the (probably unnecessary) explicit definitions and check, if the implicit definition is working.
Implicit would be preferred, if working.

Same for all the other explicit fields below

Reply:
unit tests succeed with the default implicit definition, so I deleted the explicit definitions for `description` and `title`.